go.mod

```diff
@@ -153,18 +153,15 @@ require (
 replace (
 	// TODO: fix keyring upstream
 	github.com/99designs/keyring => github.com/crypto-org-chain/keyring v1.1.6-fixes
-	github.com/cosmos/cosmos-sdk => github.com/cosmos/cosmos-sdk v0.45.2-0.20220218085826-7f949c0aa1d6
```

Comment:
not sure about if this replace is necessary with 0.42.3? @yihuang

Reply:
if there are multiple cosmos-sdk dependencies end up in go.sum, then better keep the replace I think.